azerothcore/azerothcore-wotlk
# fix(DB/Creature): Fixed Mist Howler

## Changes Proposed:
-  Added 2 new spawns
-  Changed some coords of the original 2 spawns according to sniffs
- Added roaming to all of them, added to the same pool so only 1 can be spawned and changed the spawn time to 10h.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7750

## SOURCE:


## Tests Performed:

- .go c 32884, 51885, 945802 and 945803 and check the coords
- See if he moves


## How to Test the Changes:


1. .go c 32884, 51885, 945802 and 945803 and check the coords
2. See if he moves

Test

Spawn 1
![howl spawn 1](https://user-images.githubusercontent.com/87535580/132253804-8784cf5e-81a9-4b39-a24f-fc1b59570ee0.jpg)

Spawn 2
![howl spawn 2](https://user-images.githubusercontent.com/87535580/132253806-91271bb1-66b9-47d3-87b7-1b3258018e85.jpg)

Spawn 3
![howl spawn 3](https://user-images.githubusercontent.com/87535580/132253807-34457209-23d1-40dd-8e35-f3fb9cd191da.jpg)

Spawn 4
![howl spawn 4](https://user-images.githubusercontent.com/87535580/132253815-ed4bdbc5-db58-4408-a163-d6a24f321515.jpg)

I reloaded between pictures, so we can see 2, but trust me there can be only one at the same time :P



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
